argoverse/utils/frustum_clipping.py

```diff
@@ -232,7 +231,7 @@ def generate_frustum_planes(K: np.ndarray, camera_name: str, near_clip_dist: flo
     if img_width is None or img_height is None:
         return None
 
-    P = np.array([0.0, 0.0, 0.0])
```

Comment:
i think this is probably better as an inline comment, like `# frustum starts at optical center [0,0,0]

Reply:
👍